app/src/components/misc/PersonaSurvey/ModalFooter.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { getUserPersonaSurveyDetails } from "store/selectors";
+import { actions } from "store";
+import { Col, Row } from "antd";
+import { RQButton } from "lib/design-system/components";
+import { surveyConfig } from "./config";
+import APP_CONSTANTS from "config/constants";
+import { submitAttrUtil } from "utils/AnalyticsUtils";
+import {
+  trackPersonaQ1Completed,
+  trackPersonaQ2Completed,
+  trackPersonaQ3Completed,
+  trackPersonaRecommendationSkipped,
+} from "modules/analytics/events/misc/personaSurvey";
+import "./index.css";
+
+interface FooterProps {
+  page: number;
+  fieldKey: string;
+}
+
+export const SurveyModalFooter: React.FC<FooterProps> = ({
+  page,
+  fieldKey,
+}) => {
+  const dispatch = useDispatch();
+  const surveyLength = surveyConfig.length;
+  const userPersona = useSelector(getUserPersonaSurveyDetails);
+
+  const togglePersonaSurveyModal = () => {
+    dispatch(actions.toggleActiveModal({ modalName: "personaSurveyModal" }));
+  };
+
+  const disableContinue = () => {
+    if (page === 0) return false;
+    if (userPersona[fieldKey]?.length) {
+      return false;
+    } else return true;
+  };
+
+  const renderModalLeftSection = () => {
+    if (page === 0)
+      return (
+        <>
+          <span className="survey-modal-emoji">😁</span> We are excited to see
+          you here
+        </>
+      );
+
+    if (page > 0 && page <= surveyLength - 2) {
+      return (
+        <>
+          {page}/{surveyLength - 2}
+        </>
+      );
+    } else return null;
+  };
+
+  const handleMoveToNextPage = () => {
+    switch (page) {
+      case 1:
+        trackPersonaQ1Completed(userPersona.persona);
+        submitAttrUtil(
+          APP_CONSTANTS.GA_EVENTS.ATTR.PERSONA,
+          userPersona.persona
+        );
+        break;
+      case 2:
+        trackPersonaQ2Completed(userPersona.useCase);
```

Comment:
Can we flatten the array? If we send [{}] then we won't be able to perform analysis.

```
[{ 
    value: 'Local development before the backend is ready',
    optionType: 'select'
},
{
    value: 'test'
    optionType: 'other'
}]
```
This converts to ['Local development before the backend is ready', 'OTHER: test']